sample-app-cicd/run.yaml

```diff
@@ -15,8 +14,8 @@ spec:
     - name: repo-url
       value: https://github.com/darthlukan/gitops-workshop.git
     - name: git-branch
-      value: tekton-dev
+      value: pipeline-workspaces
     - name: image-name
-      value: quay.io/btomlins/greet:v0.0.1
+      value: quay.io/sscaling/gitops-workshop:v0.0.1
```

Comment:
Using my quay.io repo because I created a robot account as per the quay.io best practices:
https://docs.quay.io/glossary/robot-accounts.html